dddwa/dddperth-website
# Updates Hero component

The design for the hero component had some changes made to it. This PR
brings the design and implementation more inline with each other.
* Updates design
* Adds format function to the dates object
* Updates hero images

## Testing
* Pull down branch
* Run in dev
* See the new hero

## Browser testing
* Chrome
* Firefox
* Safari
* iOS (Safari)
